### PR TITLE
[MIG][9.0] account_invoice_merge_purchase module

### PR DESCRIPTION
**Pull request by cubells**
- Updated README
- Updated tests

cc @Tecnativa

### CODE REVIEW

File: account_invoice_merge_purchase/README.rst [hunk 1]
```diff
@@ -1,26 +1,33 @@
...
+This module provides a compatibility between purchase order workflow and
+account invoice merge module
 
 
 Usage
 =====
 
+Select several vendor bills from the same partner and click on **Merge Partner
+Invoice** in Action button menu.
+
```
Comment (rafaelbn) [inline account_invoice_merge_purchase/README.rst:18]: Hi, I don't see this option:

**Review by rafaelbn — CHANGES_REQUESTED** [review]

**Review by chienandalu — COMMENTED** [review]
Functionality is ok. However I'd suggest to pop a warning whenever the responsible differ across invoices in the same way it does with other not compatible merging parameters (different partner, paid o open bill...).

**Comment by pedrobaeza** [comment]
@chienandalu That's out of the scope of this module

**Comment by chienandalu** [comment]
@pedrobaeza The problem is that when the bill responsible differs, those bills are not going to merge but the user has no clue about the reason why the merging didn't success. So the user is going to have to review manually every not merged bill and find out.

**Comment by pedrobaeza** [comment]
This module is only a glue module that adds purchase fields to the merge process. What you are mentioning is something for the main module, account_invoice_merge. That's why I say this is out of scope of this PR. Anyway, previous screen before merging informs about the criteria to merge invoices. You should fill an issue with the improvement you are mentioning (which is good, that's out of question), but for being done in the module account_invoice_merge.

**Comment by chienandalu** [comment]
Absolutely right, @pedrobaeza :smile: I missed the point

**Review by chienandalu — APPROVED** [review]
functionality :+1:

File: account_invoice_merge_purchase/README.rst (hunk 1 above)
Comment (pedrobaeza) [reply account_invoice_merge_purchase/README.rst:18]: Current runbot contains this.

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**